test/runtests.jl

```diff
@@ -1,6 +1,81 @@
 using Wordlegames
+using Primes
+using Tables
 using Test
 
-@testset "Wordlegames.jl" begin
-    # Write your tests here.
+const primes5 = primes(10000, 99999) # vector of 5-digit prime numbers
+const primel = GamePool(primes5)
+const primelent = GamePool(primes5; guesstype=:entropy)
+
+@testset "GamePool" begin
+    @test typeof(primel) == GamePool{5, UInt8}
+    @test length(primel.active) == 8363
+    @test eltype(primel.allscores) == UInt8
+    @test eltype(primel.targetpool) == eltype(primel.guesspool) == NTuple{5, Char}
+    @test length(first(primel.targetpool)) == 5
+    @test all(reset!(primel).active)
+    @test only(primel.guesses) == ('1', '7', '9', '2', '3')
+    @test only(primel.guessinds) == 826
+    @test only(primel.expected) ≈ 121.54167f0
+    @test only(primel.entropy) ≈ 6.62459f0
+    @test primel.guesstype == :expected
+    @test primel.hardmode
+    playgame!(primel, only(primel.guessinds))   # Got it in one!
+    @test only(primel.scores) == 0xf2
```

Comment:
I'm impressed and vaguely terrified that you used hex here :smile:

Reply:
The default printing of a `GamePool` struct in the REPL shows the scores as the `S` type and I just copied the hex representation. :shrug: